multicluster/controllers/multicluster/common/controller_utils.go

```diff
@@ -78,3 +81,49 @@ func parseServiceCIDRFromError(msg string) (string, error) {
 func NewClusterInfoResourceExportName(clusterID string) string {
 	return clusterID + "-clusterinfo"
 }
+
+func ValidateLocalClusterClaim(c client.Client, clusterSet *mcv1alpha2.ClusterSet) (clusterID ClusterID, err error) {
```

Comment:
Does it need to be public?

Reply:
Renamed it to `getClusterIDFromClusterClaim`.